### PR TITLE
1423: Add "Last Modified" column to Glossary

### LINKED ISSUE #1423: Add a column showing the date when an entry has been added to the glossary
Currently there is no way to see when a new entry has been added to the glossary. To check if a new entry has been added recently it would be great if a column is added to the glossary list showing the date when an entry was added. Then you can see quickly if the glossary has had a change.

### PR DESCRIPTION
**Pull request by pedro-mendonca**
This PR already includes the `data-sorter` for tablesorter added on https://github.com/GlotPress/GlotPress/pull/1426

- [x] Add **Last Modified** column to Glossary
- [x] Add `data-sorter` to column to set as sorteable
- [x] Add timestamp `data-text` to row data to sort alphabetically
- [x] Format data and time both in row and editor
- [x] Fix timezone offset

Fixes #1423

### CODE REVIEW

**Comment by pedro-mendonca** [comment]
This PR still requires a build step for the .min.css, should I build and add for review?

File: gp-templates/glossary-entry-row.php [hunk 1]
```diff
@@ -9,6 +9,18 @@
 		echo make_clickable( nl2br( esc_html( $entry->comment ) ) );
 		?>
 	</td>
+	<?php
+	$date_modified          = strtotime( get_date_from_gmt( $entry->date_modified ) );
+	$date_modified_formated = sprintf(
```
Comment (ocean90) [inline gp-templates/glossary-entry-row.php]:
```suggestion
	$date_modified           = strtotime( $entry->date_modified );
	$date_modified_formatted = sprintf(
```

File: gp-templates/glossary-entry-row.php [hunk 2]
```diff
@@ -9,6 +9,18 @@
...
+		/* translators: 1: Modified date. 2: Modified time. */
+		esc_html__( '%1$s at %2$s', 'glotpress' ),
+		date_i18n( get_option( 'date_format' ), $date_modified ),
+		date_i18n( get_option( 'time_format' ), $date_modified )
+	);
+	?>
+	<td class="date-modified" data-text="<?php echo esc_attr( $date_modified ); ?>">
+		<?php echo esc_html( $date_modified_formated ); ?>
```
Comment (ocean90) [inline gp-templates/glossary-entry-row.php]:
```suggestion
		<?php echo esc_html( $date_modified_formatted ); ?>
```

File: gp-templates/glossary-entry-row.php [hunk 3]
```diff
@@ -49,7 +61,7 @@
 			<h3><?php _e( 'Meta', 'glotpress' ); ?></h3>
 			<dl>
 				<dt><?php _e( 'Last Modified:', 'glotpress' ); ?></dt>
-				<dd><?php echo esc_html( $entry->date_modified ); ?></dd>
+				<dd><?php echo esc_html( $date_modified_formated ); ?></dd>
```
Comment (ocean90) [inline gp-templates/glossary-entry-row.php]:
```suggestion
				<dd><?php echo esc_html( $date_modified_formatted ); ?></dd>
```

File: gp-templates/glossary-entry-row.php [hunk 4]
```diff
@@ -9,6 +9,18 @@
 		echo make_clickable( nl2br( esc_html( $entry->comment ) ) );
 		?>
 	</td>
+	<?php
+	$date_modified          = strtotime( get_date_from_gmt( $entry->date_modified ) );
+	$date_modified_formated = sprintf(
+		/* translators: 1: Modified date. 2: Modified time. */
+		esc_html__( '%1$s at %2$s', 'glotpress' ),
+		date_i18n( get_option( 'date_format' ), $date_modified ),
+		date_i18n( get_option( 'time_format' ), $date_modified )
```
Comment (ocean90) [inline gp-templates/glossary-entry-row.php]:
```suggestion
		wp_date( get_option( 'date_format' ), $date_modified ),
		wp_date( get_option( 'time_format' ), $date_modified )
```

**Review by ocean90 — CHANGES_REQUESTED** [review]
Some minor changes related to date handling.

**Comment by pedro-mendonca** [comment]
All suggestions applied, done.

File: gp-templates/glossary-entry-row.php [hunk 5]
```diff
@@ -9,6 +9,18 @@
 		echo make_clickable( nl2br( esc_html( $entry->comment ) ) );
 		?>
 	</td>
+	<?php
+	$date_modified           = strtotime( $entry->date_modified );
+	$date_modified_formatted = sprintf(
+		/* translators: 1: Modified date. 2: Modified time. */
+		esc_html__( '%1$s at %2$s', 'glotpress' ),
```
Comment (ocean90) [inline gp-templates/glossary-entry-row.php]:
```suggestion
		__( '%1$s at %2$s', 'glotpress' ),
```

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Review by ocean90 — APPROVED** [review]

**Pull request merged**